app/helpers/group_notification_helper.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module GroupNotificationHelper
+  def group_leader_notify(data, recipient, type)
+    if type.eql? 'add_group_leader'
```

Comment:
Is there any particular reason you're using `eql?` as opposed to `==`?

Reply:
Using == came up as a warning when I ran rubocop on the commit hooks. I originally used == and changed it to conform to the rubocop checks.

As I look for information on the comparison, the rubocop style guide suggests that == is the better choice: https://github.com/rubocop-hq/ruby-style-guide

I'll make the change locally and test again before I merge.

Reply:
Oh, interesting! Either is fine then!

Reply:
I ran rubocop after changing `eql` to `==` and there were no warnings or errors, so I committed the change with ==. Maybe it was codeclimate that suggested `eql?` instead of `==`